index.bs

```diff
@@ -4939,7 +4939,7 @@ CountQueuingStrategy({ <var>highWaterMark</var> })</h5>
   1. Return *1*.
 </emu-alg>
 
-<h3 id="queue-with-sizes">Queue-with-Sizes Operations</h3>
+<h3 id="queue-with-sizes">Queue-with-Sizes operations</h3>
```

Comment:
Yeah, let's make "sizes" lowercase

Reply:
@AnthumChris Can I ask you to do this?

Reply:
@ricea All set, just pushed an `--amend` commit